lookup_plugins/oo_option.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+# vim: expandtab:tabstop=4:shiftwidth=4
+
+'''
+oo_option lookup plugin for openshift-ansible
+
+Usage:
+
+    - debug:
+      msg: "{{ lookup('oo_option', '<key> default=<default_value>') }}"
```

Comment:
Hello,
Look at my last comment. I have the feeling that a lookup plugin which fails to find what it is looking for is supposed to return an empty string.

As a consequence, if `FOO` is not set
```yaml
  - debug: msg="Env FOO={{ lookup('env', 'FOO') | default('default filter value') }}"
```
returns:
```
TASK: [debug msg="Env FOO="] ************************************************** 
ok: [localhost] => {
    "msg": "Env FOO="
}
```

if we set the boolean to True:
```yaml
  - debug: msg="Env FOO={{ lookup('env', 'FOO') | default('default filter value', True) }}"
```
we get the default value as expected:
```
TASK: [debug msg="Env FOO=default filter value"] ****************************** 
ok: [localhost] => {
    "msg": "Env FOO=default filter value"
}
```

The problem is that I cannot get empty string anymore because if I set the environment variable to `""` (empty string), it will be replaced by the `default` filter.

In short, it is not possible to distinguish between a lookup failure and a successful lookup that found a key the value of which is empty.
As a consequence, if we have
```yaml
lookup('…') | default('my_default_value', True)
```
It is not possible to get `''` empty string anymore.

If we have the default value handled by the `oo_option` plugin, it is possible to override a non-empty default value by setting the environment variable to empty string.

What are your feelings?